e2e/etcd_test.go

```diff
@@ -433,9 +433,9 @@ func (ep *etcdProcess) Stop() error {
 }
 
 func (ep *etcdProcess) waitReady() error {
-	readyStrs := []string{"enabled capabilities for version", "published"}
+	readyStrs := []string{"starting server...", "published"}
```

Comment:
As I know that the kubernetes e2e can run with a running cluster. The cluster may have a lower version than the code. Not sure they plan to do this or not. But seems it works.
In this pullreq I am trying to do the same thing for etcd e2e test. And for a basic function like waitReady maybe it is better to use some more common string for checking. And the request of checking the process is guaranteed to negotiate up to the 3.0 api or not should be in the tests or other functions.

But it is OK for me to keep it the original way as this is not the main purpose of the pullreq.